components/server/src/user/stripe-service.ts

```diff
@@ -93,27 +92,6 @@ export class StripeService {
         return priceInformation.humanReadableDescription;
     }
 
-    async cancelSubscriptionForUser(userId: string) {
```

Comment:
How is this now handled? I may have missed it in the large diff

Reply:
there are no subscriptions for users

Reply:
Sorry, missed the fact this method is "user" specific.. Makes sense